optuna/integration/sklearn.py

```diff
@@ -29,14 +36,14 @@
     _import_error = e
     _available = False
 
-from optuna import distributions
-from optuna import exceptions
-from optuna import logging
+if _available:
+    if _sklearn_version >= '0.22.1':
+        from sklearn.utils.validation import _check_fit_params as _sklearn_check_fit_params  # NOQA
+    else:
+        from sklearn.utils.validation import _index_param_value as _sklearn_index_param_value  # NOQA
```

Comment:
[question] In my understanding, two `# NOQA` are used to pass flake8 check of too long line. Is this correct?

Reply:
You're right.

Reply:
Thanks. Could you take a look at https://github.com/himkt/optuna/pull/1?